test/t/test_perldoc.py

```diff
@@ -9,7 +9,11 @@ def test_1(self, completion):
         assert "fixtures/" not in completion  # Our fixtures/ dir
         assert not [x for x in completion if "File::File::" in x]
 
-    @pytest.mark.complete("perldoc -", require_cmd=True)
+    @pytest.mark.complete(
+        "perldoc -",
+        require_cmd=True,
+        skipif="perldoc 2>/dev/null; (( $? == 1 ))",
```

Comment:
This feels a bit specific to me, especially given we're making an assumption about a perldoc that is not the real thing and could be basically whatever yet making an assumption that it'll exit with some particular status. I think I'd rather go with:
```suggestion
        skipif="! perldoc -V &>/dev/null",
```

Reply:
Makes sense, changed to that.